Frameworks/UIKit/WYPopoverController.mm

```diff
@@ -2634,8 +2639,12 @@ - (void)dismissPopoverAnimated:(BOOL)aAnimated
         }
 #endif
     };
+
+    UIPopoverPresentationController* popoverPresentationController = [viewController popoverPresentationController];
     
     void (^completionBlock)() = ^() {
+        // Keep popoverPresentationController alive via block-capture during teardown.
+        (void)popoverPresentationController;
```

Comment:
There's an implicitly strong local defined just above (ARC's enabled) but I need to keep its lifetime tied to that completionBlock via a capture. Unfortunately, I can't say do ```assert(popoverPresentationController != nil);``` in the body of the block since it can be nil in the current implementation with the deprecated UIPopoverController API. I'm inclined to leave as is but the best alternative would probably be to use ```popoverPresentationController``` in a logging statement in the block body.